cranelift/isle/veri/veri_engine/Cargo.toml

```diff
@@ -1,5 +1,7 @@
 [package]
 name = "veri_engine"
+license = "Apache-2.0 WITH LLVM-exception"
+authors = ["Alexa VanHattum, Monica Pardeshi, Michael McLoughlin, Wellesley Programming Systems Lab"]
```

Comment:
```suggestion
authors = ["Alexa VanHattum", "Monica Pardeshi", "Michael McLoughlin", "Wellesley Programming Systems Lab"]
```

Reply:
Ah, thank you! Fixing here and another spot.